src/WattleScript.Interpreter/Tree/Expressions/ExprListExpression.cs

```diff
@@ -6,7 +6,7 @@ namespace WattleScript.Interpreter.Tree.Expressions
 {
 	class ExprListExpression : Expression 
 	{
-		List<Expression> expressions;
+		internal List<Expression> expressions;
```

Comment:
this should change naming to `Expressions` instead of being lowercase

Reply:
Fixed in 4c95935184ba919fceacf7c9de78d2078e4f9abf